### PR TITLE
Make testimonials nice

### PR DESCRIPTION
**Pull request by crackeroil**
## Description
Added image functionality and overall styling to testimonials

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have added tests to cover my changes.
- [ ] All new code is fully covered by tests (see coverage report)

### CODE REVIEW

File: apps/core/models/core.py [hunk 1]
```diff
@@ -1,7 +1,17 @@
+import math
+
+from django.core.validators import ValidationError
 from django.db import models
 
 
+def validate_aspect_ratio(image):
+    ratio = 1
+    if not math.isclose(image.width / image.height, ratio, rel_tol=1e-6):
+        raise ValidationError("The aspect ratio is not correct. The aspect ratio should be: " + str(ratio))
```
Comment (bcvandendool) [inline apps/core/models/core.py:10]: Might be best to move this up to a utils file or sth, as this is now used and defined in multiple locations.

File: apps/core/static/cosmos/css/about.css [hunk 2]
```diff
@@ -44,4 +69,9 @@
 
 .partners .carousel-item {
     justify-content: center;
+}
+
+.testimonial-text {
+    font-size: 14px;
+    font-style: italic;
```
Comment (bcvandendool) [inline apps/core/static/cosmos/css/about.css:76]: I think this is possible to do with the bootstrap classes as well, which is nicer. We try to use as little manual css as possible, and use the bootstrap stuff for everything.

File: apps/core/models/core.py [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+
+
 class Testimonial(models.Model):
+    image = models.ImageField(upload_to="testimonials", blank=True, validators=[validate_aspect_ratio])
```
Comment (bcvandendool) [inline apps/core/models/core.py:14]: Make sure that it looks decent without an image as well, otherwise add a default image.

**Review by bcvandendool — CHANGES_REQUESTED** [review]

File: apps/core/models/core.py (hunk 3 above)
Comment (crackeroil) [reply apps/core/models/core.py]: I added the default image functionality, but one still needs to be uploaded to the media folder which I don't have access to.

**Review by bcvandendool — APPROVED** [review]

**Pull request merged**